app/forms/decidim/accountability/admin/result_form.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Accountability
+    module Admin
+      # This class holds a Form to create/update results from Decidim's admin panel.
+      class ResultForm < Decidim::Form
```

Comment:
This one might be tricker to inherit from :S. Still, do you think that's possible?

Reply:
Yes, I'm trying a few things.